crates/stackable-versioned-macros/src/codegen/container/struct/k8s.rs

```diff
@@ -429,43 +466,22 @@ impl Struct {
             )
                 -> ::std::result::Result<::std::vec::Vec<#serde_json_path::Value>, #convert_object_error>
             {
+                let desired_api_version = #version_enum_ident::try_from_api_version(desired_api_version)?;
+                // let desired_api_version: #version_enum_ident = desired_api_version.try_into()?;
```

Comment:
I guess this can go now:

```suggestion
```

Reply:
Sure thing! I didn't push the change :facepalm: 95a10ae